frontend/src/app/users/[id]/page.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
```

Comment:
Please install this extension and format your code before committing,
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

You can even configure VSCode to format on every save